hazelcast/src/main/java/com/hazelcast/client/impl/ClientEngineImpl.java

```diff
@@ -377,7 +373,7 @@ public void shutdown(boolean terminate) {
             try {
                 final Connection conn = endpoint.getConnection();
                 if (conn.isAlive()) {
-                    conn.close("Shutdown of ClientEngine", null);
+                    conn.close("Stopping ClientEngine", null);
```

Comment:
Wording change: Stopping -> Resetting

Reply:
I can't make it resetting because it is also called by shutdown. I will extract a method to pass correct messages here.